docs/contracts/validators-exit-bus-oracle.md

```diff
@@ -7,3 +7,341 @@ A contract implementing an on-chain message bus between the protocol off-chain o
 with the main goal of delivering validator exit requests to the Lido-participating node operators.
 
 The requests should be processed timely according to the ratified [Lido on Ethereum Validator Exits Policy V1.0](https://snapshot.org/#/lido-snapshot.eth/proposal/0xa4eb1220a15d46a1825d5a0f44de1b34644d4aa6bb95f910b86b29bb7654e330).
+
```

Comment:
Please outline a purpose and flow for this contract :pray:

Reply:
done